### ISSUE TITLE
Set up frontend transpilers (possibly gulp runner for the whole thing)

### ISSUE THREAD

**Issue by therebelrobot** [therebelrobot/brigadehub#242]
#### Description

We need to be able to split out internal js and import external libraries in a sane way, and a way to have those files watched and reloaded on change. SCSS is currently being transpiled on route load, which is not a great way of going about it.

Maybe setting gulp up with a `build` task, which handles SCSS and JS transpiling, and run that along with nodemon and watch for changes? I can take a look at this later this week if no one has the bandwidth.

**Comment by therebelrobot** [comment]
@jaydurant @Juxtaposeidon This is the issue covering what you two are coordinating on. Feel free to use this for discussion.

**Comment by Juxtaposeidon** [comment]
Hey @jaydurant  I need some help with designing the gulp workflows for css. I've perused some SASS info sites since I don't have much experience with CSS precompilers. From what I gathered, it goes something like this?
1. Make a watch task for changes to the .scss file.
2. Run exec cmd 'sass css/main.scss:css/main.css --sourcemap'   I guess this would update the .map file to accommodate the changes made file?

And that's it? I guess the /*# sourceMappingURL=main.css.map */ at the bottom of the .css file remains static.
